<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBEditorMain</class>
 <widget class="QMainWindow" name="CBEditorMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CBEditorMain</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>741</width>
      <height>572</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>737</width>
       <height>568</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="gridGraphics">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="backgroundBrush">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabTools">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>191</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tabSprites">
     <attribute name="title">
      <string>Sprite</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabObjects">
     <attribute name="title">
      <string>Objects</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="acCreateNewMap"/>
    <addaction name="acLoadMap"/>
    <addaction name="acSave"/>
    <addaction name="acSaveAs"/>
    <addaction name="separator"/>
    <addaction name="acCloseMap"/>
    <addaction name="separator"/>
    <addaction name="acExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="acEditShowGrid"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="acResizeGrid"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="acAboutMe"/>
    <addaction name="acAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="acCreateNewMap">
   <property name="text">
    <string>Create New Map..</string>
   </property>
  </action>
  <action name="acLoadMap">
   <property name="text">
    <string>Load Map..</string>
   </property>
  </action>
  <action name="acSave">
   <property name="text">
    <string>Save..</string>
   </property>
  </action>
  <action name="acSaveAs">
   <property name="text">
    <string>Save As..</string>
   </property>
  </action>
  <action name="acExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="acEditShowGrid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Grid..</string>
   </property>
  </action>
  <action name="acResizeGrid">
   <property name="text">
    <string>Resize Map</string>
   </property>
  </action>
  <action name="acAboutMe">
   <property name="text">
    <string>Me?</string>
   </property>
  </action>
  <action name="acAboutQt">
   <property name="text">
    <string>About Qt...</string>
   </property>
  </action>
  <action name="acCloseMap">
   <property name="text">
    <string>Close Map</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
